roles/os_hardening/tasks/minimize_access.yml

```diff
@@ -80,3 +80,175 @@
     fstype: proc
     opts: '{{ proc_mnt_options }}'
     state: mounted
+
+- name: Mount /boot with hardened options
+  mount:
+    path: /boot
+    src: '{{ os_mnt_boot_src }}'
+    fstype: '{{ os_mnt_boot_filesystem }}'
+    opts: '{{ os_mnt_boot_options }}'
+    state: present
+  register: bootmount
+  when: os_mnt_boot_enabled | bool
+
+- name: Harden permissions for /boot directory
+  file:
+    dest: /boot
+    owner: 'root'
+    group: 'root'
+    mode: '{{ os_mnt_boot_dir_mode }}'
```

Comment:
We should add the when-condition to all tasks, otherwise it could lead to unexpected surprises by users.
```
   when: os_mnt_boot_enabled | bool
``

Reply:
I think, that is no good idea (with this variable), because I asked there for "is /boot mounted". If we add a "when" to the harden directory task, you disable harden the directory, if there is no mountpoint, too.

We always want to harden the directory instead of introducing a new variable to enable or disable it. If the user do not want to harden the directory, the user can configure a more open mode for `/boot`. What do you think?

Reply:
@schurzi, same problem as other comment. What do you think?

Reply:
I think we are mixing different things here. In this PR all the `os_mnt_*_enabled` options are dedicated to manage mounting of separate devices for our hardening. What you are proposing @rndmh3ro is adding a control to disable hardening for a specific directory altogether, am I right?

I think we could do this by using the `os_mnt_*_enabled` as a condition for every task and additionally checking the facts dictionary for existing mount points for the specific mount points in the mount tasks. But this might be over-complicating things.

Reply:
Oh ok. So a new variable with `disable_hardening_all_mountpoints` would be a solution? I can add it to all mountpoints as a separate "when" statement.

As you wrote: I add a bool to to every mountpoint, that are optional mountpoints (everything could be on one filesystem). All other mountpoints should be managed by this hardening module.